### ISSUE TITLE
`ERROR: LoadError: type CodeInfo has no field linetable` on Julia's master branch

### ISSUE THREAD

**Issue by jishnub** [JuliaDebug/JuliaInterpreter.jl#616]
Precompilation fails with
```julia
  1 dependency had output during precompilation:
┌ JuliaInterpreter
│  ERROR: LoadError: type CodeInfo has no field linetable
│  Stacktrace:
│    [1] getproperty(ci::Core.CodeInfo, s::Symbol)
│      @ Base ./deprecated.jl:330
│    [2] linetable(arg::Core.CodeInfo)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/utils.jl:269
│    [3] JuliaInterpreter.FrameCode(scope::Method, src::Core.CodeInfo; generator::Bool, optimize::Bool)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/types.jl:148
│    [4] prepare_framecode(method::Method, argtypes::Any; enter_generated::Bool)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:178
│    [5] prepare_framecode
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:138 [inlined]
│    [6] prepare_call(f::Any, allargs::Vector{Any}; enter_generated::Bool)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:249
│    [7] prepare_call
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:235 [inlined]
│    [8] determine_method_for_expr(expr::Expr; enter_generated::Bool)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:582
│    [9] determine_method_for_expr
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:573 [inlined]
│   [10] enter_call_expr(expr::Expr; enter_generated::Bool)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:621
│   [11] enter_call_expr
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:619 [inlined]
│   [12] macro expansion
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/construct.jl:237 [inlined]
│   [13] _precompile_()
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/precompile.jl:7
│   [14] top-level scope
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/packagedef.jl:168
│   [15] include(mod::Module, _path::String)
│      @ Base ./Base.jl:559
│   [16] include(x::String)
│      @ JuliaInterpreter ~/.julia/packages/JuliaInterpreter/KozUS/src/JuliaInterpreter.jl:1
│   [17] top-level scope
│      @ ~/.julia/packages/JuliaInterpreter/KozUS/src/JuliaInterpreter.jl:14
│   [18] include
│      @ ./Base.jl:559 [inlined]
│   [19] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
│      @ Base ./loading.jl:2679
│   [20] top-level scope
│      @ stdin:4
│  in expression starting at /home/jishnu/.julia/packages/JuliaInterpreter/KozUS/src/packagedef.jl:168
│  in expression starting at /home/jishnu/.julia/packages/JuliaInterpreter/KozUS/src/JuliaInterpreter.jl:1
│  in expression starting at stdin:4
└  
```
on v"1.12.0-DEV.179"

**Comment by Zentrik** [comment]
Presumably because #606 hasn't merged yet.

**Comment by vtjnash** [comment]
Closed by #606
